### PR TITLE
(CONT-1212) - Fixing CI pipeline for Windows-2019

### PR DESCRIPTION
**Pull request by Ramesh7**
## Summary

Fixing CI Pipeline failures for Win-2019.

## Additional Context
- N/A

## Related Issues (if any)
Win-2019 need reboot after enabling `Web-Server` feature.

## Checklist
- [x] 🟢 Spec tests.
- [x] 🟢 Acceptance tests.
- [x] Manually verified. (For example `puppet apply`)

### CODE REVIEW

File: spec/spec_helper_acceptance_local.rb [hunk 1]
```diff
@@ -11,11 +11,51 @@ class LitmusHelper
...
-    LitmusHelper.instance.run_shell('Install-WindowsFeature -name Web-Server -IncludeManagementTools') unless ENV['TARGET_HOST'].nil? || ENV['TARGET_HOST'] == 'localhost'
+    unless ENV['TARGET_HOST'].nil? || ENV['TARGET_HOST'] == 'localhost'
+      LitmusHelper.instance.run_shell('Install-WindowsFeature -name Web-Server -IncludeManagementTools')
+      LitmusHelper.instance.run_shell('Install-WindowsFeature -Name Web-HTTP-Errors')
+      result = LitmusHelper.instance.run_shell('(Enable-WindowsOptionalFeature -Online -FeatureName IIS-WebServerRole -NoRestart).RestartNeeded')
+
+      if result.stdout.split("\r\n").last == 'True'
+        puts 'VM need reboot, hence doing force reboot'
+        LitmusHelper.instance.run_shell('Restart-Computer -Force')
+        sleep(300)
```
Comment (jordanbreen28) [inline spec/spec_helper_acceptance_local.rb]: Looks like redundant use of sleep. `retry_on_error_matching` is used below to wait for the VM to come back online, so no need for both.
 Can probably remove this line `sleep(300)`
Just curious on your thoughts here

File: metadata.json [hunk 2]
```diff
@@ -17,8 +17,6 @@
     {
       "operatingsystem": "Windows",
       "operatingsystemrelease": [
-        "2012",
-        "2012 R2",
```
Comment (jordanbreen28) [inline metadata.json]: I don’t think we can drop support whilst agent still supports these platforms.
If it is the case I think we should at least have this in a separate PR so it has a dedicated change log entry/release note.

Comment (Ramesh7) [reply metadata.json]: I have removed it as we don't have image available, here is from nightly build :
```
Warning: Cannot find image for Windows-2012
Warning: Cannot find image for Windows-2012 R2
```

File: spec/spec_helper_acceptance_local.rb (hunk 1 above)
Comment (Ramesh7) [reply spec/spec_helper_acceptance_local.rb]: The idea was I don't wanted to immediately try to run command as system just shutdown and its gonna take time to get it back, so having upfront 5 min sleep make sense and then we can have small retries like 5 sec with 60 or etc.

File: metadata.json (hunk 2 above)
Comment (jordanbreen28) [reply metadata.json]: Yeah I think like you noted we need to align our support, or atleast find suitable images to run the tests on

**Review by jordanbreen28 — APPROVED** [review]

**Pull request merged**